packages/api/src/platforms/vtex/resolvers/product.ts

```diff
@@ -196,4 +196,6 @@ export const StoreProduct: Record<string, Resolver<Root>> & {
     specificationGroups,
   releaseDate: ({ isVariantOf: { releaseDate } }) => releaseDate ?? '',
   advertisement: ({ isVariantOf: { advertisement } }) => advertisement,
+  deliveryPromisesBadges: ({ isVariantOf: { deliveryPromisesBadges } }) =>
```

Comment:
There are still some places where we can remove the "s", I'll comment them

```suggestion
  deliveryPromiseBadges: ({ isVariantOf: { deliveryPromisesBadges } }) =>
```

Reply:
thanksss! <3